books/RayTracingTheNextWeek.html

```diff
@@ -94,10 +93,35 @@
 </div>
 
 
+Managing Time
+--------------
+Before continuing, let's think about time, and how we might manage it across one or more renders.
+There are two aspects of shutter timing to think about: the time from one shutter opening to the
+next shutter opening, and how long the shutter stays open for each frame. Standard movie film used
+to be shot at 24 frames per second. Modern digital films can be 30 or even 60 frames per second.
+Each frame can have its own shutter speed, which need not be -- and typically isn't -- the maximum
```

Comment:
it's rare that you'll have one frame be a different shutter speed from an adjacent frame if they're from the same take.

So, "Each frame can have its own shutter speed" is a bit overkill, but it's useful-sorta-right, rather than misleading-sorta-right